From ca9b50889acd8a4c6c86ea83903b92d920945a77 Mon Sep 17 00:00:00 2001
From: wangshuobin <w14767780617@163.com>
Date: Wed, 21 Jan 2026 17:27:07 +0800
Subject: [PATCH] new-ext4-xattr-disable-credits-check

Signed-off-by: wangshuobin <w14767780617@163.com>
---
 fs/ext4/xattr.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/fs/ext4/xattr.c b/fs/ext4/xattr.c
index 2f93e8b..d0e5584 100644
--- a/fs/ext4/xattr.c
+++ b/fs/ext4/xattr.c
@@ -2327,10 +2327,6 @@ ext4_xattr_set_handle(handle_t *handle, struct inode *inode, int name_index,
 						   flags & XATTR_CREATE);
 		brelse(bh);
 
-		if (jbd2_handle_buffer_credits(handle) < credits) {
-			error = -ENOSPC;
-			goto cleanup;
-		}
 		WARN_ON_ONCE(!(current->flags & PF_MEMALLOC_NOFS));
 	}
 
-- 
2.33.0

